conda/cli/conda_argparse.py

```diff
@@ -84,6 +86,14 @@ def generate_parser():
 
 
 def do_call(args, parser):
+    """
+    Serves as the primary entry point for commands referred to in this file and for
+    all registered plugin subcommands.
+    """
+    # First, check if this is a plugin subcommand; if this attribute is present then it is
```

Comment:
To be clear, `args` here is a `Namespace` object?

Reply:
Yes.

Reply:
Most places you see `args` in conda, it is indeed a `Namespace` object.